terraform/app_service.tf

```diff
@@ -59,6 +59,10 @@ resource "azurerm_linux_web_app" "main" {
     "REQUESTS_CONNECT_TIMEOUT" = "${local.secret_prefix}requests-connect-timeout)",
     "REQUESTS_READ_TIMEOUT"    = "${local.secret_prefix}requests-read-timeout)",
 
+    # Django Azure Email Backend
+    "AZURE_COMMUNICATION_CONNECTION_STRING" = "${local.secret_prefix}${local.azure_communication_connection_string_name})",
+    "DEFAULT_FROM_EMAIL"                    = "${local.secret_prefix}${local.azure_communication_from_email_name})",
```

Comment:
Does this need to be a secret? We are basically hardcoding it in the `email.tf` anyway. I think we can just pass this in as an env var (it isn't sensitive).

Reply:
It may be a different story during `apply`... but let's try it! (You could try the `apply` locally too, against `dev`, to see)